### PR TITLE
Better exceptions.

### PR DESCRIPTION
**Pull request by SkyToGround**
As the title says.

### CODE REVIEW

**Comment by umithardal** [comment]
Great! Is there a chance these changes will help with the current problems at Ymir?

**Comment by SkyToGround** [comment]
@umithardal  Yes, I think/hope so.

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Version changes
 
+## Version 1.1.2
+
+* Slightly better exceptions when unable to connect to a broker.
```
Comment (umithardal) [inline CHANGES.md:5]: Thanks for providing change log!

File: file_writer_control/WorkerFinder.py [hunk 2]
```diff
@@ -125,5 +126,10 @@ class WorkerFinder(WorkerFinderBase):
     def __init__(self, command_topic_url: str):
         temp_cmd_ch = CommandChannel(command_topic_url)
         command_url = KafkaTopicUrl(command_topic_url)
-        temp_producer = KafkaProducer(bootstrap_servers=[command_url.host_port])
+        try:
+            temp_producer = KafkaProducer(bootstrap_servers=[command_url.host_port])
```
Comment (umithardal) [inline file_writer_control/WorkerFinder.py:130]: Unnecessary empty line :)

**Review by umithardal — APPROVED** [review]

File: file_writer_control/WorkerFinder.py (hunk 2 above)
Comment (umithardal) [reply file_writer_control/WorkerFinder.py:130]: Hmm. May be there is no empty line? In review view there was :)

Comment (SkyToGround) [reply file_writer_control/WorkerFinder.py:130]: I can see no empty line there in the actual source file

**Pull request merged**